src/MegaPoker.sol

```diff
@@ -78,8 +78,10 @@ contract MegaPoker {
         if ( univ2unieth.pass())   univ2unieth.poke();
         if (univ2wbtcdai.pass())  univ2wbtcdai.poke();
         if (univ2aaveeth.pass())  univ2aaveeth.poke();
+        if (univ2daiusdt.pass())  univ2daiusdt.poke();
 
         spot.poke("ETH-A");
+        spot.poke("ETH-B");
```

Comment:
I think the ordering here is based on chronology, not sure if we want to break that convention.

Reply:
Reverted to keep this PR moving, but I don't think it matters here.